website_event_membership_restriction/views/event_templates.xml

```diff
@@ -17,9 +17,13 @@
             expr="//form//div[@id='o_wevent_tickets_collapse']//button"
             position="after"
         >
+            <t
+                t-set="is_registration_disabled"
```

Comment:
This is still too generic... why not something like is_disabled_for_non_members

Reply:
thanks